### DISCUSSION TITLE
Alpha 176

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#6217] [Release Notes]
* Fixed search
* Fixed macOS ARM build
* From this build on, updates will be downloaded from S3 instead of Github, which should solve the issues with a wrong arch installer getting downloaded (ARM vs x64)

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.176'>Alpha 176</a>.</em>

**Comment by RockyLXH** [comment]
Hi, this version and 175 cannot choose some fonts which can be chosen in 174. like FiraCode Nerd...
